pallets/liquidity-pools/src/mock.rs

```diff
@@ -0,0 +1,143 @@
+use cfg_primitives::{PoolId, TrancheId};
+use cfg_traits::Millis;
+use cfg_types::{
+	domain_address::{Domain, DomainAddress},
+	permissions::PermissionScope,
+	tokens::{AssetStringLimit, CurrencyId, CustomMetadata, TrancheCurrency},
+};
+use frame_support::derive_impl;
+use orml_traits::parameter_type_with_key;
+use sp_runtime::{traits::IdentityLookup, AccountId32, DispatchResult, FixedU64};
+
+use crate::pallet as pallet_liquidity_pools;
+
+pub type Balance = u128;
+pub type AccountId = AccountId32;
+pub type Ratio = FixedU64;
+
+frame_support::construct_runtime!(
+	pub enum Runtime {
+		System: frame_system,
+		Time: cfg_mocks::time::pallet,
+		Permissions: cfg_mocks::permissions::pallet,
+		Pools: cfg_mocks::pools::pallet,
+		AssetRegistry: cfg_mocks::asset_registry::pallet,
+		ForeignInvestment: cfg_mocks::foreign_investment::pallet,
+		Gateway: cfg_mocks::outbound_queue::pallet,
+		DomainAddressToAccountId: cfg_mocks::converter::pallet::<Instance1>,
+		DomainAccountToDomainAddress: cfg_mocks::converter::pallet::<Instance3>,
```

Comment:
Why do we have to use `Instance3` here? IIRC we used maximum 2 instances in tests.

Reply:
Right! I had an `Instance2` before. I will change it! Good eye 😆